---
layout: page
title: "About Me"  
permalink: /about-me/  
icon: fas fa-info-circle
order: 5
---


---
Hello! My name is Jason, and I am currently a Bioinformatics Analyst in the Integrative Cardiovascular Metabolism and Pathophysiology (iCaMP) Laboratory at the Boston University Chobanian & Avedisian School of Medicine. My work focuses on applying bioinformatics and statistical techniques to explore the role of mitochondrial genetics in cardiovascular disease, particularly within large cohorts such as the Framingham Heart Study, UK Biobank, and the All of Us Study.

Previously, I served as a Post-Baccalaureate IRTA Research Fellow at the National Heart, Lung, and Blood Institute. During this time, I contributed to several projects, including a study on platelet reactivity and arterial stiffness measures, and a Transcriptome-Wide Association Study (TWAS) on heavy menstrual bleeding.

Beyond my professional pursuits, I am passionate about water quality monitoring and watershed conservation. I volunteer with OARS, a non-profit organization committed to protecting and monitoring the Sudbury, Concord, and Assabet Rivers and their tributaries. Through this volunteer work, Iâ€™ve gained hands-on experience with water quality monitoring techniques and developed a strong dedication to local sustainability and conservation efforts.

My combined experiences in bioinformatics and environmental conservation reflect my desire to apply computational tools not only to human health but also to environmental protection and sustainability.
 
 ---

## Education

- **Boston University Graduate School of Arts & Sciences, Boston, MA**
    - Master of Science, Bioinformatics, December 2024
- **College of the Holy Cross, Worcester, MA**
    - Bachelor of Arts, Biology, Minor in Creative Writing, May 2021

---

## Research Experience

### Integrative Cardiovascular Metabolism and Pathophysiology (iCaMP) Laboratory
**Boston University Chobanian & Avedisian School of Medicine**, Boston, MA  
**Data Analyst II**, January 2024-Present  
Mentors: Jessica L. Fetterman, PhD, FAHA and Deepa M. Gopal, MD, MS

- Adapt and use a bioinformatic pipeline to identify mitochondrial genetic variants in whole genome sequencing datasets
- Design and implement statistical models to analyze longitudinal and cross-sectional associations of mitochondrial genetic variants with ECG measures, and prevalent and incident atrial fibrillation in the Framingham Heart Study and other large cohorts, including All of Us (NIH)
- Develop and apply statistical models to assess interactions of genetic variants in mitochondrial genes across both the mitochondrial and nuclear genomes with circulating metabolite signatures, echocardiographic and cardiac MRI traits, and heart failure subtypes
- Apply statistical techniques to analyze metabolomic signatures in post-mortem cardiac samples from donors with hypertension and normotension
- Co-developed and deployed an interactive web application to facilitate access to proteins associated with mitochondrial function, based on a publication by Leyfer and Fetterman (doi.org/10.1093/nargab/lqad107): [MitoProximal Protein Database](https://mitoproximalproteindatabase.pythonanywhere.com)

---

### National Heart, Lung, and Blood Institute, National Institutes of Health  
**The Framingham Heart Study**, Framingham, MA  
**Post-Baccalaureate IRTA Research Fellow**, June 2021-July 2023  
Mentor: Andrew D. Johnson, PhD, FAHA

- Published a research project in *Platelets* that found trends for associations of platelet reactivity with arterial calcium burden, aortic diameter, and vascular stiffness among selected Framingham Heart Study cohorts ([PMID: 37609998](https://pubmed.ncbi.nlm.nih.gov/37609998))
- Conducted a Transcriptome-Wide Association Study (TWAS) on heavy menstrual bleeding among top GWAS meta-analysis results involving >34,000 cases and >351,000 controls of European ancestry
- Performed assays measuring platelet reactivity in response to various agonists in plasma samples from >200 participants

---

### College of the Holy Cross, Worcester, MA  
**Biology Research Assistant**, September 2020-May 2021  
Mentor: Robert Bellin, PhD

- Prepared media and solutions, maintained 3T3 fibroblast and NMuMG epithelial cell lines, and constructed EpiChip microfluidic mammary duct models for experimental use
- Developed a protocol with lab members to grow fibroblast and epithelial cell cultures in a hydrogel matrix within the EpiChip microfluidic platform, mitigating the experimental difficulties of using a typical collagen matrix
- Used the EpiChip microfluidic platform protocol that I developed with the team to study the role

---

## Skills

### Data Science and Bioinformatics Skills 
- Python, R, Git, Docker, ImageJ, FUSION (TWAS and RWAS tools suite),
Galaxy (bioinformatics analysis platform)

### Programming Skills 
- HTML, Javascript, Cascading Style Sheets (CSS), MySQL, Bash

### GIS and Mapping Skills
- ArcGIS Pro

### Laboratory Skills  
- PCR Amplification, Dot Blot Assay, Light Transmission Aggregometry, Optimul Platelet
Aggregation Assay, Light Microscopy, Phase Contrast Microscopy

--- 

<!-- ---

## Timeline

<div align="center">
{% include jekyll-timeline.html 
   startYear=2010 
   timelineHeight=600 
   col1Title="My Experience"
   col1Events=page.timelineEvents 
%}
</div>

--- -->
